### ISSUE TITLE
Please be so kind to make generator/yield stars on the same side.

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5-dev#204]
As @pjasiun wrote:

> I would go with after for both, so:
```js
function* generator() {
  yield* other();
}
```
> Star changes the behavior of function or yield, it should be part of it.

> yield *other() looks like * beeing part of the function name, like yield _other().

So the desired eslint settings would be:
```json
"generator-star-spacing": [
	"error",
	"before"
],
"yield-star-spacing": [
	"error",
	"before"
],
```

**Comment by Reinmar** [comment]
You sure about 'before' ;)?

**Comment by jodator** [comment]
yeah... [you're right](http://eslint.org/docs/rules/yield-star-spacing#after) sir
```json
"generator-star-spacing": [
	"error",
	"after"
],
"yield-star-spacing": [
	"error",
	"after"
]
```

**Comment by Reinmar** [comment]
Released in 1.0.5.
